utils/cdk-runner/pkg/aws/cloudformation/deploy.go

```diff
@@ -68,10 +69,18 @@ func DeployStack(c *Client, stackName, template string) error {
 		return err
 	}
 
-	if err := hooks.PreChangeSetApplyHook(hooks.PreChangeSetApplyHookExecutable); err != nil {
+	if err := hooks.RunChangesetHook(hooks.PreChangeSetApplyHookExecutable); err != nil {
 		return err
 	}
 
+	if err := hooks.RunChangesetHook(hooks.DryRunHookExecutable); err != nil {
+		return err
+	}
```

Comment:
Given that these two hooks share common args I've renamed the function to be generic and used by both